client/src/components/app.js

```diff
@@ -3,21 +3,36 @@ import $ from 'jquery';
 import TitleLeft from './titleLeft';
 import TitleRight from './titleRight';
 import MapBox from './mapBox';
-import ModalShare from './modals/modalShare';
-import photoCarousel from '../images/photoCarousel.png';
 import Modal from './modals/modal';
+import ModalShare from './modals/modalShare';
+import ModalSave from './modals/modalSave';
+import ModalMap from './modals/modalMap';
+import ModalDetails from './modals/modalDetails';
+import ModalSend from './modals/modalSend';
 
+import photoCarousel from '../images/photoCarousel.png';
+import yelpHeader from '../images/img-yelp-header.png';
 
 class App extends React.Component {
   constructor() {
     super();
 
     this.state = {
       business: null,
-      showModal: false,
+      // showModal: false,
+      showModalDetails: false,
```

Comment:
Since these are all part of the modal, maybe have a single Modal component and inside that divide it up into smaller components? This way your import routes are easier to understand.

Reply:
That's a good idea. I had the intention of refactoring my modals to reduce code duplication. Will refactor.